src/main/java/org/jenkinsci/test/acceptance/po/WorkflowJob.java

```diff
@@ -37,4 +47,20 @@ public WorkflowJob(Injector injector, URL url, String name) {
     public final Control script = control("/definition/script");
     public final Control sandbox = control("/definition/sandbox");
 
+    public String copyResourceStep(String filePath) {
+        final Resource res = resource(filePath);
+        try (InputStream in = res.asInputStream()) {
```

Comment:
It's not marked as `@NonNull` but [it never returns null](https://github.com/jenkinsci/acceptance-test-harness/blob/c8a96001366835afd3d8fa15e4e6ed0dfd611443/src/main/java/org/jenkinsci/test/acceptance/po/CapybaraPortingLayerImpl.java#L431), but throws `AssertionError` if the resource does not exist.